Framework/SINQ/test/PoldiFitPeaks2DTest.h

```diff
@@ -60,10 +61,8 @@ class PoldiFitPeaks2DTest : public CxxTest::TestSuite {
 
   void testSetDeltaTFromWorkspace() {
     MatrixWorkspace_sptr ws = WorkspaceCreationHelper::create2DWorkspace(1, 10);
-    for (size_t i = 0; i <= 10; ++i) {
-      ws->dataX(0)[i] = static_cast<double>(i);
-    }
-
+    auto &X = ws->mutableX(0);
+    std::iota(X.begin(), X.end(), 0);
```

Comment:
const iterators? Can't this be rewritten as `ws->setBinEdges(0, ws->x(0).size(), LinearGenerator(0, 1))` although I'm not sure it's worth introducing the includes, and the current code seems clearer